projects/components/src/overlay/overlay.service.ts

```diff
@@ -32,7 +32,8 @@ export class OverlayService {
         type: PopoverPositionType.Fixed,
         location: config.position ?? PopoverFixedPositionLocation.RightUnderHeader
       },
-      data: metadata
+      data: metadata,
+      backdrop: config.backdrop ?? PopoverBackdrop.None
```

Comment:
How are we handling backdrop styling?

Reply:
Users have three options, they can pass in:
- `PopoverBackdrop.None`
- `PopoverBackdrop.Opaque`
- `PopoverBackdrop.Transparent`

The opaque background applies the default backdrop from the Material library. The transparent backdrop is invisible. If we want to enable more styling options for the backdrop, we could refactor this to allow developers to pass in their own CSS class name.